lib/v8-log-to-ticks.js

```diff
@@ -9,9 +9,9 @@ const pump = require('pump')
 
 module.exports = v8LogToTicks
 
-function v8LogToTicks (isolateLogPath) {
+function v8LogToTicks (isolateLogPath, node) {
   const isJson = extname(isolateLogPath) === '.json'
-  const sp = isJson || spawn(process.argv[0], [
+  const sp = isJson || spawn(node, [
```

Comment:
This should be the one passed, not the one 0x is using itself. The reason is that we are somewhat confident that a Node.js release could process its own data, but not across multiple release lines.

Reply:
That is exactly what my change does.